lib/components/Input/NumberInput.tsx

```diff
@@ -111,15 +111,13 @@ export class NumberInput extends React.Component<NumberInputProps, NumberInputSt
             !this.isPositive()
             /** Firefox uses a different keycode for dashes (-) than other browsers... */
             && (event.keyCode === keyCode.dash || event.keyCode === keyCode.firefoxDash)
```

Comment:
please add a comment to docs about FF, e.g. http://www.javascripter.net/faq/keycodes.htm